app/views/va_facilities/show.html.erb

```diff
@@ -0,0 +1,61 @@
+<% provide :main_classes, 'bg-gray-2' %>
+<% provide :footer_classes, 'bg-gray-2' %>
+
+<% provide :head_tags do %>
+  <%= javascript_include_tag 'facility_directory', 'data-turbolinks-track': 'reload' %>
+<% end %>
+<%= render partial: 'va_facilities/facility_map'%>
+<section class="usa-section padding-y-0 grid-col-12">
+  <div class="grid-container position-relative grid-col-12">
+    <div class="grid-row grid-gap">
+      <div class="grid-col-12">
+        <h1 class="font-sans-2x1 desktop:margin-top-5 desktop:margin-bottom-3">
+          <%= @va_facility.official_station_name %>
+        </h1>
+        <div class="grid-col-12">
+          <div class="grid-col-6" style="float: left;"> This facility has created <%= @practice_results_count %> practices and has adopted <%= @adoptions_count %> practices.  It's a<% if @va_facility.rurality == "U" || @va_facility.rurality == "I" %>n<% end %>
+            <% if @va_facility.rurality == "R" %>
+              rural
+            <% end %>
+            <% if @va_facility.rurality == "U" %>
+              urban
+            <% end %>
+            <% if @va_facility.rurality == "H" %>
+              highly rural
+            <% end %>
+            <% if @va_facility.rurality == "I" %>
+              isolated
+            <% end %>
```

Comment:
Can this be an `if` `else` statement instead of a bunch of `if`s? Or a `switch case`.